qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1520A.py

```diff
@@ -27,6 +27,7 @@ class B1520A(B1500Module):
             class.
         slot_nr: Slot number of this module (not channel number)
     """
+    time_out_phase_compensation = 300
```

Comment:
is there no way to calculate/predict this value? if not at all, please make a note about it in the example notebook.

Reply:
There is no way to predict this. Manual says that phase compensation takes around 30 seconds. To be on the safe side I have put timeout to be 60 s. I have added the note in example notebook and also an inline comment.